packages/web-app-files/src/components/SideBar/Audio/AudioMetaPanel.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <div id="files-sidebar-panel-audio">
+    <dl class="audio-data-list">
+      <template v-if="title">
+        <dt v-text="$gettext('Title')" />
+        <dd v-text="title" />
+      </template>
+      <template v-if="duration">
```

Comment:
I like the idea to only show the available data. But I think while showing the exif/meta data, it makes more sense to actually show the existing and non existing data e.G with a dash(-). 

Could you check how the mac is handling missing data ?

Reply:
Mac Finder is only showing the fields that were found. That's why I built it as well. But no hard feelings, I'll make a screenshot with `-` for missing fields and then we can discuss.

Reply:
<img width="442" alt="Screenshot 2024-05-29 at 10 30 40" src="https://github.com/owncloud/web/assets/3532843/b00e2044-1a18-4d30-b35a-20b57a54f750">
<img width="443" alt="Screenshot 2024-05-29 at 10 32 21" src="https://github.com/owncloud/web/assets/3532843/6f25da54-c1dc-4b6e-ac17-ff3b9378f444">

Reply:
I kind a like it that way

Reply:
I think it looks a bit weird when you have hardly any data in there - but no strong opinion either. I'm happy either way that this is finally happening after I started metadata extraction it last year :)

Reply:
No strong opinion either from my side. I'd probably hide the fields without data, but I can still see the point in having them 🤷